packages/config/config/devices/0x0115/haseman_r4d4.json

```diff
@@ -0,0 +1,726 @@
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"67": {
+			"label": "Channel 4 power up memory",
+			"description": "When Power Up memory is enabled, the module will save the actual status of the associated channel output in case of power break",
+			"valueSize": 2,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"minValue": 0,
+			"maxValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"68": {
+			"label": "Channel 5 power up memory",
+			"description": "When Power Up memory is enabled, the module will save the actual status of the associated channel output in case of power break",
+			"valueSize": 2,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"minValue": 0,
+			"maxValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"69": {
+			"label": "Channel 6 power up memory",
+			"description": "When Power Up memory is enabled, the module will save the actual status of the associated channel output in case of power break",
+			"valueSize": 2,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"minValue": 0,
+			"maxValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"70": {
+			"label": "Channel 7 power up memory",
+			"description": "When Power Up memory is enabled, the module will save the actual status of the associated channel output in case of power break",
+			"valueSize": 2,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"minValue": 0,
+			"maxValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"71": {
+			"label": "Channel 8 power up memory",
+			"description": "When Power Up memory is enabled, the module will save the actual status of the associated channel output in case of power break",
+			"valueSize": 2,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"minValue": 0,
+			"maxValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disabled",
+					"value": 0
+				},
+				{
+					"label": "Enabled",
+					"value": 1
+				}
+			]
+		},
+		"72": {
+			"label": "Channel 1 button type",
+			"valueSize": 2,
+			"defaultValue": 1,
+			"readOnly": false,
+			"writeOnly": false,
+			"minValue": 1,
+			"maxValue": 4,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Push button",
+					"description": "Short press is switching On/Off, Long press is Dimming / Brightening, Double press is directly switching to MAX LEVEL",
```

Comment:
We don't support descriptions for options. Merge them into a single description for the parameter.